ifaddr/_posix.py

```diff
@@ -82,4 +88,4 @@ def add_ip(adapter_name, ip):
 
     libc.freeifaddrs(addr0)
 
-    return ips.values()
+    return list(ips.values())
```

Comment:
No changing runtime behavior please, please don't change the return type in a PR that's supposed to document the *existing* behavior.

Reply:
I will change this back, however, leaking implementation details needs to be addressed at some point.

Reply:
It may or may not need to be addressed – it we document this return types as `Iterable` and explicitly mention in the documentation that static type hints can be depended on while the actual types returned (which can be narrower) shouldn't, I think we're fine here.